Formula/deno.rb

```diff
@@ -0,0 +1,56 @@
+class Deno < Formula
+  desc "Command-line JavaScript / TypeScript engine"
+  homepage "https://deno.land/"
+  url "https://github.com/denoland/deno.git",
+    :tag      => "v0.6.0",
+    :revision => "22feb74ba12215597416b5531f8a557302283e79"
+
+  depends_on "llvm" => :build
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build # GN require Python 2.7+
+  depends_on "rust" => :build
+
+  # https://bugs.chromium.org/p/chromium/issues/detail?id=620127
+  depends_on :macos => :el_capitan
+
+  resource "gn" do
+    url "https://gn.googlesource.com/gn.git",
+      :revision => "64b846c96daeb3eaf08e26d8a84d8451c6cb712b"
+  end
+
+  def install
+    # Build gn from source, move it to the expected location and add it to the PATH
+    (buildpath/"gn").install resource("gn")
+    cd "gn" do
+      system "python", "build/gen.py"
+      system "ninja", "-C", "out/", "gn"
+    end
+    (buildpath/"third_party/v8/buildtools/mac").install buildpath/"gn/out/gn"
+    ENV.prepend_path "PATH", buildpath/"third_party/v8/buildtools/mac"
```

Comment:
Just to clarify, these are the two lines that may change in the future if upstream(s) listen to the feedback discussed here, correct?